content/en/docs/ops/ambient/usage/ztunnel/_index.md

```diff
@@ -0,0 +1,498 @@
+---
+title: Ztunnel Networking - L4 & M-TLS
+description: User guide for ztunnel networking.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: n/a
```

Comment:
The code snippets have been tested on a live cluster. Since the current guide is alpha only and we want to get it out to assist early testers of ambient it would be better to get the first pass of this guide merged without adding the tests. Also some of the snippets involve trimmed output and may change in the second pass of updates before ambient goes to beta.  We can add the tests on a followup update to this guide as it approaches beta status. This was okay with the other reviewers, if needed we can discuss further, else for the first pass, we would like to merge wirthout the tests of the code snippets.